views/update/review-beneficial-owner-individual.html

```diff
@@ -47,8 +47,27 @@ <h1 class="govuk-heading-xl govuk-!-margin-0">Review the individual beneficial o
       <input type="hidden" name="dob_year" value="{{ date_of_birth.year }}"/>
       <input type="hidden" name="dob_month" value="{{ date_of_birth.month }}"/>
 
-      <input type="hidden" name="date_of_birth" value="{{ date_of_birth }}"/>
+      {% if populateAddress %}
```

Comment:
why conditional?

Reply:
Residential address will be private data in review of BO, but should be populated if the user clicks the change link. This will only be set when the change link is clicked